src/sanitizer.js

```diff
@@ -198,7 +198,8 @@ const INVALID_INLINE_STYLE_REGEX =
  * @return {string}
  */
 export function sanitizeHtml(html) {
-  const tagPolicy = htmlSanitizer.makeTagPolicy();
+  const tagPolicy = htmlSanitizer.makeTagPolicy(parsed =>
+    parsed.scheme_ === 'https' ? parsed : null);
```

Comment:
do parsed.getScheme(), as scheme_ is a private field and it is best to access it via its API.

Reply:
done.